README.md

```diff
@@ -175,8 +175,17 @@ conversation.conversation_parts[1].body
 # REPLYING TO CONVERSATIONS
 # User (identified by email) replies with a comment
 intercom.conversations.reply(:id => conversation.id, :type => 'user', :email => 'joe@example.com', :message_type => 'comment', :body => 'foo')
-# Admin (identified by email) replies with a comment
-intercom.conversations.reply(:id => conversation.id, :type => 'admin', :email => 'bob@example.com', :message_type => 'comment', :body => 'bar')
+# Admin (identified by id) replies with a comment
+intercom.conversations.reply(:id => conversation.id, :type => 'admin', :admin_id => '123', :message_type => 'comment', :body => 'bar')
+
+# Open
+intercom.conversations.open(id: conversation.id, admin_id: '123')
+
+# Close
+intercom.conversations.close(id: conversation.id, admin_id: '123')
+
+# Assign
+intercom.conversations.assign(id: conversation.id, admin_id: '123', assignee_id: '124')
```

Comment:
i think we made a call to leave these out of ruby client (and maybe go) for now. cc @josler